packages/css/src/heading/heading.scss

```diff
@@ -22,39 +20,23 @@
 }
 
 .amsterdam-heading--1 {
-  font-size: var(--amsterdam-heading-1-narrow-font-size);
-  line-height: var(--amsterdam-heading-1-line-height);
-
-  @media screen and (min-width: $amsterdam-breakpoint-typography) {
-    font-size: var(--amsterdam-heading-1-wide-font-size);
-  }
+  font-size: var(--amsterdam-heading-density-low-level-1-font-size);
+  line-height: var(--amsterdam-heading-density-low-level-1-line-height);
```

Comment:
High density is to be implemented for all three components updated so far. This will use the approach with the local custom properties that we have in grid.